README.md

```diff
@@ -40,41 +40,54 @@ On RedHat you get the necessaries with:
 
 On Ubuntu / Debian with:
 
-    apt-get install perl unzip
+    apt-get install perl unzip autoconf carton
```

Comment:
hmmm autoconf is only required when building the git checkout ... carton should never be required

Reply:
Hmm... I always started with bootstrap (unlike the README instructions) so indeed autoconf may be an overshot. Apparently this (both) is also what @Dacesilian used on Gitter.

No opinion about carton, except that maybe using an OS-provided one is somehow better for support/compat/etc?

Reply:
Curious, the release zip file does not include `configure` script, but the tarball does.
It is both tracked in git SCM and excluded in `.gitignore`, if that helps.

Reply:
Amended the README to convey that bootstrap exists and then might need autoconf and family to be used.